dandi/dandiapi.py

```diff
@@ -1071,6 +1076,31 @@ def iter_upload_raw_asset(
             self, metadata=asset_metadata, jobs=jobs, replacing=replace_asset
         )
 
+    def has_data_standard(self, data_standard: str) -> bool:
+        """
+        Returns True if the Dandiset contains one or more files of the indicated
+        standard. Otherwise, returns False. This is determined by checking for
+        the RRID of the standard  in the "dataStandard" field of the assetsSummary of
+        the dandiset.
+
+        :param data_standard: str that can either be "NWB", "BIDS", or an RRID of a
+        standard.
+        :type data_standard: str
+        """
+        if data_standard in DATA_STANDARD_MAP:
+            rrid = DATA_STANDARD_MAP[data_standard]
+        elif data_standard.startswith("RRID:"):
+            rrid = data_standard
+        else:
+            raise ValueError(
+                f"'data_standard' must be an RRID (of form 'RRID:XXX_NNNNNNN`) or one "
+                f"of the following values: {", ".join(DATA_STANDARD_MAP.keys())}"
+            )
+        assets_summary = self.get_raw_metadata()["assetsSummary"]
```

Comment:
well, `assetsSummary` seems to be non optional, so must be present in a valid metadata record, but if not we would get `KeyError` here... which could be good as to alert user that smth is wrong with metadata in that dandisets.  But should it be an exception really or just a warning?  I would lean to a be a warning and just return `False`. WDYT @jwodder and @satra ?

Reply:
@jwodder and @bendichter WDYT - better be strict or robust?

Reply:
I'd be fine with changing this to throw a warning if assetsSummary is missing